lib/tasks/accounts/update_adopter_status.rake

```diff
@@ -4,33 +4,33 @@ namespace :accounts do
   task update_adopter_status: [:environment] do
     last_id = nil
     loop do
-      contacts = OpenStax::Salesforce::Remote::Contact.select(
-        :id,
-        :adoption_status,
-        :accounts_uuid
-      )
-      .where("Accounts_UUID__c != null")
-      .order(:Id)
-      .limit(250)
-      contacts = contacts.where("Id > '#{last_id}'") unless last_id.nil?
-      contacts = contacts.to_a
-      last_id = contacts.last&.id
+      users = User.where(adopter_status: nil).order(:id).limit(250)
```

Comment:
I would say `User.where(adopter_status: nil).where.not(salesforce_contact_id: nil).limit(250)` here (order(:id) is also fine but optional in this case).